src/asmcnc/comms/localization.py

```diff
@@ -102,6 +129,17 @@ def load_from_dictionary(self):
                     self.dictionary[str(lines[self.default_lang])] = str(lines[self.lang])
             log("Loaded language in from full dictionary")
 
+            # For Korean characters to show up, an external font is required
+            if self.lang == "Korean (KO)":
+                self.font_regular = self.korean_font
+                self.font_bold = self.korean_font_bold
+            else:
+                # Roboto is the standard kivy font
+                self.font_regular = self.standard_font
+                self.font_bold = self.standard_font_bold
+
+            Builder.load_string(builder_font_string % (self.font_regular, self.font_bold))
```

Comment:
This simply updates the kivy Widget class to use these two fonts, so that all instances use them from this point onwards. Then any widget that inherits from this class (so every widget in kivy) will be created with this font. Resulting in the software automatically being built in the chosen font.